engine-rest/engine-rest-openapi/src/main/templates/lib/utils.ftl

```diff
@@ -21,8 +22,15 @@
       </#if>
 
       "type": "${type}"
-      <#if format?has_content>
-      ,"format": "${format}"
+
+      <#if type == "array">,
```

Comment:
awesome!

Reply:
let's mention this in the openapi readme

Reply:
I will document this in a separate commit in the main ticket (CAM-11485)